src/search/FindBar.js

```diff
@@ -705,8 +705,11 @@ define(function (require, exports, module) {
      */
     FindBar.getInitialQuery = function (currentFindBar, editor) {
         var query,
-            selection = FindBar._getInitialQueryFromSelection(editor),
+            selection = "",
```

Comment:
nit clean: Please use ternary operator, 
`selection = editor ? FindBar._getInitialQueryFromSelection(editor) : "";`

Reply:
Thanks for pointing it out. Forgot to add the ternary operator. Corrected.